steps/google-play-rollout-update/0.0.1/step.yml

```diff
@@ -0,0 +1,57 @@
+title: Google Play Rollout Update
```

Comment:
Based on the Step Title conventions: `Update Google Play Rollout` (or something like this) would be better: https://devcenter.bitrise.io/en/steps-and-workflows/developing-your-own-bitrise-step/developing-a-new-step.html#the-title